index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="assets/stylesheets/main.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+  <p class="welcome">Welcome</p>
+  <nav class="mobile-menu"><i class="fa fa-bars"></i></nav>
```

Comment:
- [ ] Kindly use header tag mainly for site's title or logo (or both) and the main site navigation. Implement this after:
- The logo or title should be inserted into 'a' tag. This is for three reasons: convention, semantics, and user experience(UX).